include/deal.II/non_matching/mapping_info.h

```diff
@@ -179,6 +182,49 @@ namespace NonMatching
           }
       }
     };
+
+    template <int dim, int spacedim = dim>
+    dealii::internal::MatrixFreeFunctions::GeometryType
+    compute_geometry_type(
```

Comment:
Is this copy/paste from the matrix-free code?

Reply:
Copied and adapted. We should aim to reuse this piece of code. Let me see if I can make it fit in MatrixFree MappingInfo as well.